### PR TITLE
briefs search test suite fixes

### PR DESCRIPTION
**Pull request by vadik0101**

### CODE REVIEW

File: src/components/CustomModal.tsx [hunk 1]
```diff
@@ -14,7 +14,11 @@ const CustomModal = ({
   className,
 }: CustomModalProps) => {
   return open ? (
-    <div onClick={onClose} className={`${className}`}>
+    <div
+      data-testid="filter-modal"
+      onClick={onClose}
+      className={`${className}`}
```
Comment (cuteolaf) [inline src/components/CustomModal.tsx:20]: replace `${className}` with className

File: tests/__mocks__/briefsData.ts [hunk 2]
```diff
@@ -186,6 +186,38 @@ export const intermediateExpData = [
...
+    id: 3,
+    headline: "briefThree",
+    industries: [
+      {
+        id: 23,
+        name: "finance three",
+      },
+    ],
+    description: "finance brief three",
+    skills: [
```
Comment (cuteolaf) [inline tests/__mocks__/briefsData.ts:200]: update skills

**Review by cuteolaf — CHANGES_REQUESTED** [review]

File: src/components/CustomModal.tsx (hunk 1 above)
Comment (vadik0101) [reply src/components/CustomModal.tsx:20]: fixed

File: tests/__mocks__/briefsData.ts (hunk 2 above)
Comment (vadik0101) [reply tests/__mocks__/briefsData.ts:200]: okay

**Pull request merged**